src/app/models/propTypes/featureIndex/index.js

```diff
@@ -0,0 +1,22 @@
+import { arrayOf, bool, shape, string, number } from 'prop-types';
+import { cpsMetadataPropTypes } from '../metadata';
+import groupsPropTypes from '../frontPageGroup';
+import relatedContentPropTypes from '../frontPageRelatedContent';
```

Comment:
I am getting this warning and error messages when running `npm run dev`:

![error-warning](https://user-images.githubusercontent.com/46446236/67267119-94867180-f4a9-11e9-92b9-d2ce3b0bba5b.png)

I believe the import should be `cpsFrontPageMetadataPropTypes` instead of `cpsMetadataPropTypes`, because it is the one exported in `propTypes/metadata/index.js`, and also:

```suggestion
import relatedContentPropTypes from '../relatedContent';
```

Reply:
I'll take a look!